assets/src/styles/blocks/CarouselHeader/CarouselHeaderEditorStyle.scss

```diff
@@ -69,6 +69,7 @@
     font-size: 14px;
     border: none;
     outline: none;
+    white-space: nowrap;
```

Comment:
This is a small fix for the dropdown menu in the editor, with the add/remove slide and add/edit image actions... Unrelated to the ticket but was bothering me 🙃